products/mautic.md

```diff
@@ -36,17 +36,24 @@ identifiers:
 -   purl: pkg:github/mautic/mautic/
 
 releases:
--   releaseCycle: "5.0"
-    releaseDate: 2024-01-09
+-   releaseCycle: "5.1"
+    releaseDate: 2024-09-13
     eoas: false
     eol: false
+    latest: "5.1.1"
+    latestReleaseDate: 2024-09-18
+
+-   releaseCycle: "5.0"
+    releaseDate: 2024-01-09
+    eoas: 2024-09-13
+    eol: 2024-09-13
     latest: "5.0.4"
     latestReleaseDate: 2024-04-11
 
 -   releaseCycle: "4.4"
     releaseDate: 2022-06-27
-    eoas: 2024-01-09
-    eol: 2024-04-09
+    eoas: false
+    eol: false
```

Comment:
4.4.13 was released recently, thus mark both eoas and eol as false.

Reply:
I don't agree with this change. On https://www.mautic.org/mautic-releases you can clearly see that 4.4 is EOL. Looks like the Mautic team decided to make an exception for this release, but it should not change the officially documented support and EOL dates.

Reply:
okay, let me raise an upstream issue about this (they should update that release page as well per the recent release)

Reply:
aha ![image](https://github.com/user-attachments/assets/110042ca-4b5d-446b-8669-bad27c990648)